partner_email_check/tests/test_partner_email_check.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 Komit <https://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from unittest.mock import patch
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import SavepointCase
+from odoo.tools.misc import mute_logger
+
+
+class TestPartnerEmailCheck(SavepointCase):
```

Comment:
@JuanyDForgeflow @LoisRForgeFlow Could you change this to TransactionCase ?

Reply:
@rousseldenis @yankinmax I just changed it